snap/snapcraft.yaml

```diff
@@ -1,36 +1,56 @@
-name: saber
-version: 0.9.4
-summary: Handwritten notes
-description: A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and macOS!
+name: saber # you probably want to 'snapcraft register <name>'
+base: core18 # the base snap is the execution environment for this snap
```

Comment:
May I ask why you've chosen `core18`?

The docs ([here](https://snapcraft.io/docs/base-snaps)) say
> `core22` is the currently recommended base for the majority of snaps. (...)

Reply:
Because cores above core18 don't support flutter plugin, and I'm not succesful in manually compiling your app under core22 and gnome extension. If you can go for it. 

https://snapcraft.io/docs/supported-plugins